Islandora/islandora_ocr
# Adding escaped quotes surrounding term

**ISLANDORA-2518**: (https://jira.lyrasis.org/browse/ISLANDORA-2518)

# What does this Pull Request do?
It appears that search terms such as those with certain apostrophes can lead to xpath warnings.
This solves the warnings produced: "Invalid predicate in HOCR->findTermNodes()" and "Invalid expression in HOCR->findTermNodes()"

# How should this be tested?

A description of what steps someone could take to:
* Add the file attached to the JIRA ticket as the HOCR datastream of a page object.
* Navigate to the object and add the following url params:
```
?solr[query]=bird's eye&solr[params][defType]=dismax&solr[params][sort]=score desc&solr[params][facet]=true&solr[params][facet.mincount]=1&solr[params][facet.limit]=100&solr[params][facet.field][1]=mods_genre_ms&solr[params][facet.field][2]=mods_relatedItem_host_titleInfo_title_ms&solr[params][facet.field][3]=mods_subject_topic_ms&solr[params][facet.field][4]=mods_subject_name_personal_namePart_ms&solr[params][facet.date][0]=mods_originInfo_dateIssued_mdt&solr[params][f.mods_originInfo_dateIssued_mdt.facet.date.start]=NOW/YEAR-200YEARS&solr[params][f.mods_originInfo_dateIssued_mdt.facet.date.end]=NOW&solr[params][f.mods_originInfo_dateIssued_mdt.facet.date.gap]=%2B1YEAR&solr[params][facet.date.start]=NOW/YEAR-20YEARS&solr[params][facet.date.end]=NOW&solr[params][facet.date.gap]=%2B1YEAR&solr[params][f.mods_originInfo_dateIssued_mdt.facet.mincount]=0&solr[params][fq][0]=-RELS_EXT_isConstituentOf_uri_mt%3A[* TO *]&solr[params][fq][1]=RELS_EXT_isViewableByUser_literal_ms%3A"anonymous" OR RELS_EXT_isViewableByRole_literal_ms%3A"anonymous user" OR ((*%3A* -RELS_EXT_isViewableByUser_literal_ms%3A[* TO *]) AND (*%3A* -RELS_EXT_isViewableByRole_literal_ms%3A[* TO *]))
```
* The object should load appropriately with various highlighting in place.

# Interested parties
* @Natkeeran 
* @Islandora/7-x-1-x-committers
